CMakeLists.txt

```diff
@@ -1422,8 +1422,11 @@ if (BUILD_TESTS)
   # Tests require tools
   set(BUILD_TOOLS ON)
   enable_testing()
+  set(proc_supports_avx 0)
```

Comment:
nit: It looks like `0` does work, but we're using `OFF` for all our bools.

Reply:
Done.